### PR TITLE
Add additonal info about node placement APIs

### PR DESCRIPTION
**Pull request by tiraboschi**
Add additonal info about node placement APIs

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
All tests passed

File: docs/cluster-configuration.md [hunk 1]
```diff
@@ -35,6 +40,11 @@ See https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#aff
 * `tolerations` is a list of tolerations applied to the relevant kind of pods.
 See https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/ for more info.
 
+#### Operators placement
+The HyperConverged Cluster Operator and the operators for its component are supposed to be deployed by the Operator Lifecycle Manager (OLM).
+Thus, the HyperConverged Cluster Operator si not going to directly influence its own placement but that should be influenced by the OLM.
```
Comment (nunnatsa) [inline docs/cluster-configuration.md:45]: > Cluster Operator ***si*** not going

Cluster Operator is not going

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Pull request merged**